.editorconfig

```diff
@@ -50,4 +50,58 @@ dotnet_diagnostic.CS4014.severity = error
 dotnet_style_require_accessibility_modifiers = omit_if_default:error
 
 # CA1063: Implement IDisposable Correctly
-dotnet_diagnostic.CA1063.severity = error
\ No newline at end of file
+dotnet_diagnostic.CA1063.severity = error
+
+# Add braces (IDE0011)
+csharp_prefer_braces = true
+dotnet_diagnostic.IDE0011.severity = error
+
+# Pattern matching
+dotnet_style_object_initializer = true:suggestion
+dotnet_style_collection_initializer = true:suggestion
+dotnet_style_coalesce_expression = true:suggestion
+dotnet_style_null_propagation = true:suggestion
+dotnet_style_explicit_tuple_names = true:suggestion
+dotnet_style_prefer_is_null_check_over_reference_equality_method=true:suggestion
+
+csharp_style_pattern_matching_over_is_with_cast_check = true:suggestion
+csharp_style_pattern_matching_over_as_with_null_check = true:suggestion
+csharp_style_inlined_variable_declaration = true:suggestion
+csharp_style_throw_expression = true:suggestion
+csharp_style_conditional_delegate_call = true:suggestions
+
+# Naming rules
+
+dotnet_diagnostic.IDE1006.severity = error
+
+## Public Fields are kept Pascal Case
+dotnet_naming_symbols.public_symbols.applicable_kinds           = field
+dotnet_naming_symbols.public_symbols.applicable_accessibilities = public
+
+dotnet_naming_style.first_word_upper_case_style.capitalization = first_word_upper
+
+dotnet_naming_rule.public_members_must_be_capitalized.symbols   = public_symbols
+dotnet_naming_rule.public_members_must_be_capitalized.style    = first_word_upper_case_style
+dotnet_naming_rule.public_members_must_be_capitalized.severity = suggestion
+
+## Instance fields are camelCase
+dotnet_naming_rule.instance_fields_should_be_camel_case.severity = error
+dotnet_naming_rule.instance_fields_should_be_camel_case.symbols = instance_fields
+dotnet_naming_rule.instance_fields_should_be_camel_case.style = instance_field_style
+
+dotnet_naming_symbols.instance_fields.applicable_kinds = field
+
+dotnet_naming_style.instance_field_style.capitalization = camel_case
+dotnet_naming_style.instance_field_style.required_prefix = 
```

Comment:
Are we missing something after the equals sign? Or does leaving it blank ensure that we don't use `_` for fields?

Reply:
This is left blank to prevent the _ prefix as you suggest